WORKSPACE

```diff
@@ -4,18 +4,26 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
     name = "build_bazel_rules_nodejs",
-    sha256 = "0942d188f4d0de6ddb743b9f6642a26ce1ad89f09c0035a9a5ca5ba9615c96aa",
-    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.38.1/rules_nodejs-0.38.1.tar.gz"],
+    sha256 = "f9e7b9f42ae202cc2d2ce6d698ccb49a9f7f7ea572a78fd451696d03ef2ee116",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/1.6.0/rules_nodejs-1.6.0.tar.gz"],
 )
 
-load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories")
+load("@build_bazel_rules_nodejs//:index.bzl", "yarn_install")
 
-node_repositories(
```

Comment:
why remove this? Shouldn't we still fix the yarn/node versions used?

Reply:
This rule is states as an alternative option to having modules be managed by yarn ([source](https://bazelbuild.github.io/rules_nodejs/Built-ins.html#creating-dependency-installation-scripts-for-manually-managed-dependencies)).

After a quick scan of the documentation, there doesn't seem to be an option to specify the node version in `yarn_install`. I imagine that this could be controlled from the `package.json`? I'll keep exploring.